### PR TITLE
Application lifecycles are hard

### PR DESCRIPTION
**Pull request by designatednerd**

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/ios-developers Anyone wanna do a quick code review?

File: XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m [hunk 1]
```diff
@@ -69,6 +74,11 @@ - (id)initWithBundle:(NSBundle *)plugin
     return self;
 }
 
+- (void)applicationDidFinishLaunching:(NSNotification *)notification
+{
+    [self setupMenuItem];
```
Comment (vokal-isaac) [inline XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m]: I don't see this being removed from wherever it used to be—is there any risk of it being called twice and yielding duplicate menu items?

Comment (designatednerd) [inline XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m]: Only if `applicationDidFinishLaunching` gets called twice.

Comment (vokal-isaac) [inline XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m]: Presumably, there exist some versions of Xcode on which the existing `setupMenuItem` call was working, so won't those versions now have it run wherever that call is and then run again where you added it?

Comment (designatednerd) [inline XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m]: Ahhhhh derp. I need to remove the old one - I think that got eaten by a stash.

Comment (designatednerd) [inline XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m]: @vokal-isaac fix pushed

**Comment by vokal-isaac** [comment]
(:alien:, :alien:, :alien:, )

**Comment by designatednerd** [comment]
har har.

**Pull request merged**